vm-virtio/src/vhost_user/net.rs

```diff
@@ -68,18 +69,19 @@ impl<'a> Net {
 
         // Get features from backend, do negotiation to get a feature collection which
         // both VMM and backend support.
-        let backend_features = vhost_user_net
+        let mut backend_features = vhost_user_net
             .get_features()
             .map_err(Error::VhostUserGetFeatures)?;
         avail_features &= backend_features;
+        backend_features = avail_features;
```

Comment:
I think this one can be removed. The backend_features should remain unchanged, it represents the set of features supported by the backend.

Reply:
This line is removed.